bridge/client/app/_components/ktb-project-settings-git-extended/ktb-project-settings-git-extended.component.ts

```diff
@@ -1,32 +1,31 @@
-import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { DtRadioChange } from '@dynatrace/barista-components/radio';
 import { IGitDataExtended, IGitHttps, IGitSsh } from '../../_interfaces/git-upstream';
 import { DataService } from '../../_services/data.service';
 import { ActivatedRoute, ParamMap } from '@angular/router';
 import { filter, map } from 'rxjs/operators';
-import { isGitHTTPS } from '../../_utils/git-upstream.utils';
+import { isGitHTTPS, isGitSSH } from '../../_utils/git-upstream.utils';
 
 export enum GitFormType {
-  SSH,
-  HTTPS,
+  SSH = 'SSH',
```

Comment:
What's the reason why you assigned strings to the enum?

Reply:
Because compiler complained that SSH is not known. That solved the issue.

Reply:
For me, it is working fine without it.

Reply:
Yes you are right, also probably a leftover from the rewrites. Removed it.